source/config/profileUpgradeSteps.py

```diff
@@ -377,3 +378,21 @@ def upgradeConfigFrom_10_to_11(profile: ConfigObj) -> None:
 			"hidBrailleStandard added to braille display auto detection excluded displays. "
 			f"List is now: {profile['braille']['auto']['excludedDisplays']}",
 		)
+
+
+def upgradeConfigFrom_11_to_12(profile: ConfigObj) -> None:
```

Comment:
In the past, an upgrade step always provided for the removal of a value. That 's not the case now. How to get rid of the values in the future?

Reply:
Upgrade steps can still be used exactly the same as before. The upgrade step was added here so that the new config key is created and synchronised when the new version of NVDA is first installed. When the old key is removed, the config spec will again be incremented and removal code added.